x-pack/test/security_solution_cypress/es_archives/unmapped_fields/data.json

```diff
@@ -0,0 +1,14 @@
+{
+  "type": "doc",
+  "value": {
+    "id": "_eZE7mwBOpWiDweStB_c",
+    "index": "unmapped-7.12.0-2021.03.10-000001",
+    "source": {
+      "@timestamp":"2021-02-22T21:00:49.337Z",
+      "mydestination":{
+        "ip": ["117.242.211.13", "118.243.221.14"]
```

Comment:
I would be careful about using public facing IP's as they can end up being malicious later and scanners will pick them up in our code base and flag them.

I would use 127.0.0.1, 127.0.0.2 for example